core/render/vi/__init__.py

```diff
@@ -163,11 +163,23 @@ def index(*args, **kwargs):
 index.exposed = True
 
 
+def get_settings():
+    fields = ["admin.name", "admin.logo", "admin.login.background", "admin.login.logo", "admin.color.primary",
+              "admin.color.secondary"]
+    res = {}
+    for field in fields:
+        res[field] = conf.get(field, "")
```

Comment:
Couldn't we just load the values from the config with the `admin.` prefix? All values in the config should have a default values, which means we don't need the `.get()`.

```suggestion
     fields = {key: values for key, values in conf.items()
              if key.startswith("admin.")}
```